smdebug/tensorflow/keras.py

```diff
@@ -730,6 +730,56 @@ def _prepare_collections_for_tf2(self):
             self._unwrap_model_with_input_output_saver()
         self.prepared_tf2_collections = True
 
+    def _stop_detailed_profiling(self, current_step=None):
+        """Stop detailed profiling if the TF profiler is supported in the current TF version and detailed profiling
+        was already started.
+        """
+        if not is_profiler_supported_for_tf_version() or not self.is_detailed_profiling:
+            return
+
+        if current_step is None:
```

Comment:
Remove this `if-check` if `self.mode_steps[self.mode]` has `current_step` already.

Reply:
Done